GolangPractice-02-DataStructures/maps.go

```diff
@@ -3,14 +3,21 @@ package main
 import "fmt"
 
 func main() {
-	// To create an empty map use syntax - make(map[key-type]val-type)
```

Comment:
Why did you remove this comment?

Reply:
The comment was wrong for the syntax used, so I have deleted the comment.